demos/refactory/demo.config.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
`'refactory'` is not a descriptive name for the demo. Something like `'projection-chain'` would be more specific.